File: Java/Easy/Accepted/check-if-word-equals-summation-of-two-words_5.java
class Solution {
    public boolean isSumEqual(String firstWord, String secondWord, String targetWord) {
        
        return getNumber(firstWord) + getNumber(secondWord) == getNumber(targetWord);
    }
    
    private int getNumber(String word) {
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < word.length(); i++) {
            sb.append((int)(word.charAt(i) - 'a'));
        }
        return Integer.valueOf(sb.toString());
    }
}

File: C++/Easy/Accepted/check-if-word-equals-summation-of-two-words_5.cpp
class Solution {
public:
    bool isSumEqual(string firstWord, string secondWord, string targetWord) {
        
        return getNum(firstWord) + getNum(secondWord) == getNum(targetWord);
    }
    
    int getNum(string word) {
        int num = 0;
        for (char c : word) {
            num = num * 10 + c - 'a';
        }
        return num;
    }
};